projects/plugins/jetpack/_inc/client/state/site/reducer.js

```diff
@@ -412,25 +411,16 @@ export function hasActiveSearchPurchase( state ) {
 }
 
 /**
- * Searches active products for an active VideoPress product.
- *
- * @param {*} state - Global state tree
- * @returns {boolean} True if the an active VideoPress plan was found, false otherwise.
- */
-export function getActiveVideoPressPurchase( state ) {
-	return find( getActiveProductPurchases( state ), product =>
-		isJetpackVideoPress( product.product_slug )
-	);
-}
-
-/**
- * Determines if the site has an active VideoPress product purchase
+ * Determines if the site has an active VideoPress feature.
  *
  * @param {*} state - Global state tree
  * @returns {boolean} True if the site has an active VideoPress product purchase, false otherwise.
  */
-export function hasActiveVideoPressPurchase( state ) {
-	return !! getActiveVideoPressPurchase( state );
+export function hasActiveVideoPressFeature( state ) {
```

Comment:
Since this only abstracts two feature checks, how would you feel about keeping those inline?

Reply:
Sounds good to me. 👍